### ISSUE TITLE
Windows a11y unit tests failing. Closing Engine tree.

### ISSUE THREAD

**Issue by zanderso** [flutter/flutter#98427]
Not sure why this didn't start happening until well after https://github.com/flutter/engine/pull/31387. Not reverting anything because it looks like this needs investigation first.

**Comment by zanderso** [comment]
The failing unit test is listed in the logs as `AXPlatformNodeWinTest.UIAGetPropertySimple`.

**Comment by chinmaygarde** [comment]
We are [not getting backtraces on Windows](https://github.com/flutter/flutter/issues/98432) but I bet the fix is simple. These would make diagnosing the failures simpler.

**Comment by cbracken** [comment]
Patch is out to disable these tests in CI in flutter/engine#31452. Chrome's implementation of UI Automation support was only partially-complete at the time we forked //third_party/accessibility. We don't use UIA in Flutter, but rely instead entirely on the MSAA accessiblity implementation.

I'll take a look at getting backtraces enabled; thanks for the heads-up.

**Comment by cbracken** [comment]
The above patch has landed.
